config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -2870,6 +2844,14 @@ public HttpSecurity oidcLogout(Customizer<OidcLogoutConfigurer<HttpSecurity>> oi
 		return HttpSecurity.this;
 	}
 
+	public HttpSecurity oidcBackChannelLogout(Customizer<OidcLogoutConfigurer<HttpSecurity>> oidcBackChannelLogoutCustomizer)
```

Comment:
Please add JavaDoc so folks can read how to use the method. Please make sure the JavaDoc includes `@since 6.5`.